lib/plugins/project-json/index.js

```diff
@@ -0,0 +1,23 @@
+import processJSON from '../helper/process-json';
+import { jsonRegExKeyValue } from '../helper/project-json-regex-builder';
+import insertLink from '../../insert-link';
+
+function linkDependency(blob, key, value) {
+  const regex = jsonRegExKeyValue(key, value);
+
+  insertLink(blob.el, regex, {
+    resolver: 'nugetUrl',
+    target: '$1',
+    type: 'project_json',
+  });
+}
+
+export default class ProjectJson {
+  getPattern() {
+    return ['project.json'];
+  }
+
+  parseBlob(blob) {
+    processJSON(blob, { '$.dependencies': linkDependency });
```

Comment:
Do you think there is a need to support the `tools` field as well? 
https://github.com/aspnet/cli-samples/blob/30932a1738d27bd2567ee6790de9160f2e1e841b/HelloMvc/project.json#L29